.bintray.json

```diff
@@ -0,0 +1,20 @@
+{
+    "package": {
+        "name": "coq",
+        "repo": "coq",
+        "subject": "coq"
+    },
+
+    "version": {
+        "name": "8.7~alpha"
```

Comment:
Hum, this does not fully correspond to the proposal here: https://github.com/coq/coq/pull/483#issuecomment-306645169
@maximedenes Could you clarify the policy for naming versions? I don't understand why it was postponed for so long.